include/Network/HTTP/BadGatewayException.php

```diff
@@ -9,6 +9,7 @@
 /**
  * @brief Bad Gateway (502) http exception
  */
-class BadGatewayException extends HTTPException {
+class BadGatewayException extends HTTPException 
+{
```

Comment:
Is is this now a code-convention to put curly braces of classes/interfaces into next line?